below/procfs/src/test.rs

```diff
@@ -80,11 +81,12 @@ impl TestProcfs {
     }
 
     fn get_net_reader(&self) -> NetReader {
+        let logger = get_logger();
```

Comment:
You can just do

```
    let plain = slog_term::PlainSyncDecorator::new(std::io::stderr());
    let logger = slog::Logger::root(slog_term::FullFormat::new(plain).build().fuse(), slog::o!())
```

to avoid using `common`